src/HasStatuses.php

```diff
@@ -55,13 +55,42 @@ public function latestStatus(...$names): ?Status
         return $statuses->whereIn('name', $names)->first();
     }
 
+    /**
+     * Check if the model has ever had a status with the given name.
+     *
+     * This method determines whether the current model instance has ever had a status
+     * with the specified name.
+     *
+     * @param string $name The name of the status to check for.
+     *
+     * @return bool Returns true if the model has ever had the status with the given name,
+     *              otherwise returns false.
+     */
     public function hasEverHadStatus($name): bool
     {
         $statuses = $this->relationLoaded('statuses') ? $this->statuses : $this->statuses();
 
         return $statuses->where('name', $name)->count() > 0;
     }
 
+    /**
+     * Check if the model has never had a status with the given name.
+     *
+     * This method determines whether the current model instance has never had a status
+     * with the specified name.
+     *
+     * @param string $name The name of the status to check for.
+     *
+     * @return bool Returns true if the model has never had the status with the given name,
+     *              otherwise returns false.
+     */
+    public function hasNeverHadStatus($name): bool
+    {
+        $statuses = $this->relationLoaded('statuses') ? $this->statuses : $this->statuses();
```

Comment:
Instead of adding logic here, couldn't we just call `hasEverHadStatus` and negate the result?

Reply:
Defiantly, I will make the needed changes.

Thank you for your review.

Reply:
Hi @freekmurze,

Thank you for the valuable feedback. I've refactored the `hasNeverHadStatus` method to reuse the `hasEverHadStatus` method negated result, as suggested.